### DISCUSSION TITLE
Is it possible to create conditonal packageRules? (Update version A only if version B has a minimum version

### DISCUSSION THREAD

**Discussion by mgh87** [renovatebot/renovate#22871] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.115.2

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Desired use case
If a Spring boot parent starter is <3.x then don't upgrade logstash-encoder higher than 7.3
Is this possible?

### Context

logstash-encoder dropped logback-core 1.2 support with 7.4

#### Spring boot starter parent 2.7.12
Includes the managed dependency for logback-core 1.2
#### Spring boot starter parent 3.0.6
Includes the managed dependency for logback-core 1.4.7

### Workaround
Exclude logstash-encoder from updates for all services until all our services are Spring boot 3

### What you tried so far.

Reding the docs, searching google, asking ChatGPT and searching through this discussion board.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
I'm not aware of any way to achieve this

**Reply by mgh87** [reply]
Thanks.
